airbyte-integrations/connectors/source-bing-ads/unit_tests/test_source.py

```diff
@@ -145,3 +158,87 @@ def test_accounts(mocked_client, config):
     accounts = Accounts(mocked_client, config)
     _ = list(accounts.read_records(SyncMode.full_refresh))
     mocked_client.request.assert_called_once()
+
+
+@patch.object(source_bing_ads.source, "Client")
+def test_ad_groups(mocked_client, config):
+    ad_groups = AdGroups(mocked_client, config)
+    _ = list(ad_groups.read_records(SyncMode.full_refresh, {"campaign_id": "campaign_id"}))
+    mocked_client.request.assert_called_once()
+
+
+@patch.object(source_bing_ads.source, "Client")
+def test_ads(mocked_client, config):
+    ads = Ads(mocked_client, config)
+    _ = list(ads.read_records(SyncMode.full_refresh, {"ad_group_id": "ad_group_id"}))
+    mocked_client.request.assert_called_once()
+
+
+@patch.object(source_bing_ads.source, "Client")
+def test_campaigns(mocked_client, config):
+    campaigns = Campaigns(mocked_client, config)
+    _ = list(campaigns.read_records(SyncMode.full_refresh, {"account_id": "account_id"}))
+    mocked_client.request.assert_called_once()
```

Comment:
can you please replace it with one parametrized test?

Reply:
Agree, with @davydov-d This should be a single test with multiple inputs.